pkg/machine/ignition.go

```diff
@@ -307,7 +307,12 @@ machine_enabled=true
 	delegateConf := `[Service]
 Delegate=memory pids cpu io
 `
-	subUID := `%s:100000:1000000`
+	// Prevent subUID from clashing with actual UID
+	subUID := 100000
```

Comment:
You could also boost this number to a far larger number, Say 2,000,000,000 And then it is less likely to conflict.

Reply:
I guess it's better to make sure it doesn't conflict for certain, so I think this logic is necessary regardless